kolibri2zim/scraper.py

```diff
@@ -880,18 +932,21 @@ def sanitize_inputs(self):
         channel_meta = self.db.get_channel_metadata(self.channel_id)
 
         # input  & metadata sanitation
-        period = datetime.datetime.now().strftime("%Y-%m")
+        period = datetime.datetime.now(datetime.UTC).strftime("%Y-%m")
         if self.fname:
             # make sure we were given a filename and not a path
-            self.fname = Path(self.fname.format(period=period))
-            if Path(self.fname.name) != self.fname:
-                raise ValueError(f"filename is not a filename: {self.fname}")
+            fname_path = Path(self.fname.format(period=period)).resolve()
+            if not fname_path.is_file():
+                msg = f"filename is not a filename: {self.fname}"
+                raise ValueError(msg)
+            self.clean_fname = fname_path.as_posix()
```

Comment:
We don't need another variable just for that. Either it's useful and we use it on `self.fname` or it's not and we drop it